### ISSUE TITLE
Mod Loading Handling.

### ISSUE THREAD

**Issue by Andrey2470T** [minetest/minetest#8769]
Possibility to handle a loading of mods, e.g. in what order, what ones, reject their loading. I`d suggest to add such feature also for CSMs in order to avoid the cheating and any malicious intentions of players.

**Comment by paramat** [comment]
Why?
Mods can only load in the order defined by their dependencies, so being able to define an order doesn't seem useful, and is actually already done using dependencies.
Disabling mods is also already possible in world configuration.

**Comment by Andrey2470T** [comment]
Well, let`s say we don`t know what exact number of mods a user has. And a mod (or more than one ) is/are needed to be loaded after all other. In this situation it`s impossible to define dependencies to that mod.

**Comment by SmallJoker** [comment]
For that use-case there's `minetest.register_on_mods_loaded`. Would you like to have a similar callback for CSM?

**Comment by Andrey2470T** [comment]
This callback doesn`t fit for my case because the mod would be already loaded previously is what I don`t want. I want this to be loaded only after.

**Comment by SmallJoker** [comment]
If your mod loads first, you can modify the API functions to handle your own code first.
If your mod loads last, you can use `minetest.register_on_mods_loaded`.

Both cases have a solution. I also don't get your point of "cheating". If cheating is a problem, it must be fixed by the engine and not by mods that prevent others from loading.

**Comment by Andrey2470T** [comment]
It seems I say a nonsense. I've just understood I can use minetest.register_on_loaded_mods() in my case and that mod doesn't quite need to be loaded after. So, I close this issue.
